src/main/java/bio/terra/service/SamClientService.java

```diff
@@ -182,14 +173,18 @@ public void verifyAuthorization(
         SamClientService.ResourceType resourceType,
         String resourceId,
         SamClientService.DataRepoAction action) {
+        final String userEmail = userReq.getEmail();
+        final String institute = userEmail.substring(userEmail.indexOf("@") + 1);
+        logger.info("email: {}, action: {}, institute: {}", userEmail, action, institute);
+        // TODO add user id
```

Comment:
maybe move this logging to the isAuthorized method in the class, since it might be called instead of verifyAuthorization (when calling code wants a boolean instead of exception thrown)